numpy/__init__.pyi

```diff
@@ -434,11 +418,6 @@ class timedelta64(signedinteger):
     @overload
     def __add__(self, other: datetime64) -> datetime64: ...
     def __sub__(self, other: Union[timedelta64, int]) -> timedelta64: ...
-    if sys.version_info[0] < 3:
-        @overload
-        def __div__(self, other: timedelta64) -> float: ...
-        @overload
-        def __div__(self, other: float) -> timedelta64: ...
     @overload
```

Comment:
I think these changes conflict with an open PR

Reply:
PR https://github.com/numpy/numpy/pull/17188 includes some but not all of these.

Would you prefer that's merged first?

Or remove these changes from this PR?